internal/core/services/room_manager/room_manager.go

```diff
@@ -101,6 +101,7 @@ func (m *RoomManager) CreateRoom(ctx context.Context, scheduler entities.Schedul
 	return room, instance, err
 }
 
+// TODO(gabrielcorado): should we "force" the room status to be "Terminating"?
```

Comment:
@gabrielcorado  In this case, the runtime watcher would already update its status to terminating if the deletion succeeds, right? in this case I don't think we should be forcing this status here

Reply:
> In this case, the runtime watcher would already update its status to terminating if the deletion succeeds, right?
Yes. The only scenario where this is not true is when the runtime watcher is not running.